docs/serverless/06-01-function-cr.md

```diff
@@ -22,7 +22,7 @@ spec:
   env:
   - name: PERSON_NAME
     value: "John"
-  deps: |
+  deps: |-
```

Comment:
shouldn't that be updated in the tutorials as well?

Reply:
Ok, I will update :)

Btw: It's a `Block Chomping Indicator`

Via https://yaml-multiline.info:
> Block Chomping Indicator: The chomping indicator controls what should happen with newlines at the end of the string. The default, clip, puts a single newline at the end of the string. To remove all newlines, strip them by putting a minus sign (-) after the style indicator. Both clip and strip ignore how many newlines are actually at the end of the block; to keep them all put a plus sign (+) after the style indicator.